dinesykmeldte-backend/src/main/kotlin/no/nav/syfo/minesykmeldte/api/MineSykmeldteApi.kt

```diff
@@ -7,18 +7,27 @@ import io.ktor.routing.Route
 import io.ktor.routing.get
 import io.ktor.routing.put
 import no.nav.syfo.application.BrukerPrincipal
+import no.nav.syfo.log
 import no.nav.syfo.minesykmeldte.MineSykmeldteService
 import no.nav.syfo.minesykmeldte.model.HttpErrorMessage
 import no.nav.syfo.minesykmeldte.model.HttpMessage
 import no.nav.syfo.util.getBrukerPrincipal
 import no.nav.syfo.util.getParam
 import java.util.UUID
+import kotlin.time.ExperimentalTime
+import kotlin.time.measureTimedValue
 
+@OptIn(ExperimentalTime::class)
 fun Route.registerMineSykmeldteApi(mineSykmeldteService: MineSykmeldteService) {
     get("api/minesykmeldte") {
         val principal: BrukerPrincipal = call.getBrukerPrincipal()
         val lederFnr = principal.fnr
-        call.respond(mineSykmeldteService.getMineSykmeldte(lederFnr))
+        val timedValue = measureTimedValue {
```

Comment:
Kanskje vi kan bruke grafana til dette i stedet? Vi har vel metrikker for responstider på api-kall fra før..? Eller tenker du at vi bør vite hvor mange sykmeldte de hadde de som har de høyeste responstidene?

Reply:
Ja dette var mer en midlertidig greie for å sjekke ytelsen i dev med de 1000 sykmedlte

Reply:
vi har jo metrikker for det på APIet egentlig